plugins/hls-cabal-fmt-plugin/hls-cabal-fmt-plugin.cabal

```diff
@@ -0,0 +1,43 @@
+cabal-version:      2.4
+name:               hls-cabal-fmt-plugin
+version:            0.1.0.0
+synopsis:           TODO
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             The Haskell IDE Team
+copyright:          The Haskell IDE Team
+maintainer:         jana.chadt@nets.at
+category:           Development
+build-type:         Simple
+extra-source-files: LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.CabalFmt
+  hs-source-dirs:   src
+  build-depends:
+    , base            >=4.12 && <5
+    , cabal-fmt
+    , filepath
+    , ghcide          >=1.2  && <1.5
+    , hls-plugin-api  ^>=1.1
+    , lsp-types
+    , process
+    , text
+    , transformers
+
+  default-language: Haskell2010
+
+test-suite tests
+  type:             exitcode-stdio-1.0
+  default-language: Haskell2010
+  hs-source-dirs:   test
+  main-is:          Main.hs
+  ghc-options:      -threaded -rtsopts -with-rtsopts=-N
+  build-depends:
+    , base
+    , filepath
+    , hls-cabal-fmt-plugin
+    , hls-test-utils        ^>=1.0
```

Comment:
Could you update dependencies? For example, I think this one actually does not work with GHCIDE 1.2 (or any version before this patch), as it requires `whemUriHaskellFile`.

Reply:
Could you please elaborate? I don't understand :see_no_evil:

Reply:
Code-changes in GHCIDE are actually not related to what the plugin needs to successfully compile. Is it still sensible to bump the lower version? Admittedly, this plugin will malfunction with older versions, but it is theoretically compatible.

Reply:
@fendor, with that malfunctioning, should we call this plugin compatible? I mean, it will be compiled, but if the plugin does not work well, what's the point of the compiling?